app/models/workflow_execution.rb

```diff
@@ -23,6 +23,14 @@ def prepared?
     state == 'prepared'
   end
 
+  def submitted?
+    state == 'submitted'
+  end
+
+  def cancelled?
```

Comment:
This should be `canceled?` as well as `state == 'canceled'` This will need to be updated in the service as well.

Reply:
Decided to leave this as is

Reply:
Canadian spelling vs American spelling. I'm not sure if there's a reason to prefer one over the other in code. Maybe @ericenns has input?